omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PersonResource1_8.java

```diff
@@ -50,7 +50,7 @@
  * {@link Resource} for Person, supporting standard CRUD operations
  */
 @Resource(name = RestConstants.VERSION_1 + "/person", order = 2, supportedClass = Person.class, supportedOpenmrsVersions = {
-	"1.8.*", "1.9.*", "1.10.0 - 1.10.3" })
+        "1.8.*", "1.9.*", "1.10.0 - 1.10.3" })
```

Comment:
Do we need these changes?

Reply:
@dkayiwa none of those were intended, surprised to find them too